### ISSUE TITLE
Implement simple CI

### ISSUE THREAD

**Issue by mabartos** [mabartos/keycloak-adaptive-authn#15]
+ Simple CI for PRs to verify everything works as expected
+ At least build the project

**Comment by lezhdz98** [comment]
I'm new doing contributions to open-source projects, I would like to try this issue.

**Comment by mabartos** [comment]
@lezhdz98 Yeah, sure. As there are no tests yet, it'd be fine to at least build the project and verify there's no compilation error. Thank you!

**Comment by lezhdz98** [comment]
@mabartos, I did the CI for the building, but there is an error with the dependencies.

Could you please show me how are you building the project on your local machine?

I tried these commands and there are errors

mvn -B package --file pom.xml   

./mvnw clean install -DskipTests  

mvn clean install -DskipTests -Dkeycloak.version=999.0.0
